KCP CLI: rename to kcp, documentation (#270)
kyma-project/control-plane: docs/cli/commands/kcp.md

# kcp

> Day-two operations tool for Kyma Runtimes

Reviewer: ```suggestion
Day-two operations tool for Kyma Runtimes.
```

## Synopsis

> The KCP CLI (Kyma Control Plane CLI) is a day-two operations tool for Kyma Runtimes, which allows you to view and manage the Runtimes in scale.

Reviewer: ```suggestion
KCP CLI (Kyma Control Plane CLI) is a day-two operations tool for Kyma Runtimes which allows you to view and manage the Runtimes in scale.
```

> It is possible to list and observe attributes and state of each Kyma Runtime and perform various operations on them, such as upgrading the Kyma version.

Reviewer: ```suggestion
It is possible to list and observe attributes and state of each Kyma Runtime, and perform various operations on them, such as upgrading the Kyma version.
```

You can find the complete list of possible operations as commands below.

> The CLI supports configuration file for common, global options needed for all commands. The config file will be looked up in this order:

Reviewer: ```suggestion
The CLI supports configuration file for common (global) options needed for all commands. The config file will be looked up in this order:
```

  - `--config {PATH}` option
  - KCPCONFIG environment variable which contains the path
  - $HOME/.kcp/config.yaml (default path).

> The configuration file is in YAML format and supports the following global options: oidc-issuer-url, oidc-client-id, oidc-client-secret, keb-api-url, kubeconfig-api-url, gardener-kubeconfig.

Reviewer: ```suggestion
The configuration file is in the YAML format and supports the following global options: `oidc-issuer-url`, `oidc-client-id`, `oidc-client-secret`, `keb-api-url`, `kubeconfig-api-url`, `gardener-kubeconfig`.
```

Author: Unfortunately this part is generated from CLI global options list, and I can't easily apply Markdown syntax here.

> Please see the Global Flags / Options section for the description of these options.

Reviewer: ```suggestion
See the **Global Options** section of each command for the description of these options.
```

---

> title: kcp

Author: I will move all detailed contents to here from usage. overview and usage will remain just really some starting page to those generated documents.
